antora.yml

```diff
@@ -1,6 +1,6 @@
 name: server
 title: ownCloud Server
-version: master
+version: '10.1'
```

Comment:
Sorry, I have pushed that fix directly to the branch by accident, please drop this change.

Reply:
I just pulled the latest version of 10.1 and don't see it there.

Reply:
Ah, I see it now, in the conflict with `master`.

Reply:
No need to quote it... Just drop the change ;)

Reply:
I'm keeping the `version` value in this branch, as it should work as a reminder to quote the value supplied. I set it to `10.0` for the 10.0 branch and the fraction was truncated.

Reply:
Where? I haven't seen such a behavior locally.

Reply:
I saw it happen when I was creating the patch/PR for 10.0 branch.